### DISCUSSION TITLE
Suggestions on updating and adding new functions

### DISCUSSION THREAD

**Discussion by qyls8268** [RandomEngy/VidCoder#750] [Ideas]
Hello, developer, vidcoder software, can add subtitle track to export SRT, I think this is a very practical function - a user from China.这是一款不错的软件，谢谢开发者。

**Comment by RandomEngy** [comment]
VidCoder exposes the functionality of HandBrake. To get that feature, you'd need to ask the [HandBrake team](https://handbrake.fr/community.php) to add it.

**Reply by qyls8268** [reply]
I can't visit Facebook in China, could you contact handbrake team and ask it to add subtitle track export function? Thank you very much for your help.

**Reply by RandomEngy** [reply]
https://forum.handbrake.fr/

**Comment by jontext88** [comment]
A useful function would be a Video passthrough. It exists for audio but a Video passthrough would enable a fast file processing when only the audio needs re-encoding.

**Reply by RandomEngy** [reply]
VidCoder uses the HandBrake API to do its encoding. Unfortunately the HandBrake team has said that remuxing/direct video copy is not something they intend to add. You could always [ask them to add it](https://handbrake.fr/community.php).

**Comment by mpek** [comment]
Handbrake support moved feature requests to Github: https://github.com/HandBrake/HandBrake/issues

**Reply by RandomEngy** [reply]
Thanks, updated my saved reply templates with that link.
